core/stat/base/leap_array.go

```diff
@@ -210,7 +210,7 @@ func (la *LeapArray) valuesWithTime(now uint64) []*BucketWrap {
 	if now <= 0 {
 		return make([]*BucketWrap, 0)
 	}
-	ret := make([]*BucketWrap, 0)
+	ret := make([]*BucketWrap, 0, 8)
```

Comment:
maybe `la.array.length` is better?

Reply:
Good advice. Length of result mostly is same to  `la.array.length`.